blog/src/postsConstructor.js

```diff
@@ -0,0 +1,9 @@
+import { format } from 'date-fns'
+import uuid from 'react-uuid'
+
+function PostConstructor(text){
+    this.text = text
+    this.date = format(new Date(), 'MM/dd/E/yyyy HH:mm')
+    this.id = uuid()
+}
```

Comment:
```suggestion
const getPost = (text) => ({
    text: text,
    date: format(new Date(), 'MM/dd/E/yyyy HH:mm'),
    id: uuid(),
});
```
нет сымсла создавать конструктор постов если можно просто возвращать объект